### ISSUE TITLE
Import on Swift gives error 'No such module'

### ISSUE THREAD

**Issue by Yozine** [bluebillywig/playerSdkLibraries#5]
Using the BlueBillywigPlayerSDK using cocoapods version 1.10.1 gives a ' no such module 'BlueBillywigPlayerSDK' on import.

Podfile contents:

```
# platform :ios, '9.0'

target 'TestProject' do
  use_frameworks!
... 
...
  pod 'BlueBillywigPlayerSDK'
end
 
```

I perform a pod install on this project which succeeds. The following prompt shows in my console:

```
Analyzing dependencies
Downloading dependencies
Installing BlueBillywigPlayerSDK (1.4.1.31)
Generating Pods project
Integrating client project
Pod installation complete! There are 2 dependencies from the Podfile and 2 total pods installed.

```

In a view controller of the same target I call:
import BlueBillywigPlayerSDK

It gives me the following error:
_No such module 'BlueBillywigPlayerSDK'_

As a work-around I've ressorted to pinning down the version to 1.4.1.21
_pod 'BlueBillywigPlayerSDK', '= 1.4.1.21'_

I'd like to be able to use the latest changes, what can I do to solve this issue?

**Comment by jhildensperger** [comment]
I believe this issue started after version 27 https://github.com/bluebillywig/playerSdkLibraries/tree/1.4.1.27

**Comment by fgroenendijk** [comment]
Hello @Yozine,

Thanks for bringing this issue to our attention.

Coming sprint we will work to fix this issue.
We will inform you when it's fixed.

With kind regards,

Floris Groenendijk

**Comment by fgroenendijk** [comment]
We've fixed this in version 1.4.1.32 which just went live.
